src/sentry/api/urls.py

```diff
@@ -1546,6 +1547,11 @@
                     OrganizationRelayUsage.as_view(),
                     name="sentry-api-0-organization-relay-usage",
                 ),
+                url(
+                    r"^(?P<organization_slug>[^\/]+)/functions/$",
+                    OrganizationSentryFunctionEndpoint.as_view(),
+                    name="sentry-api-0-organization-functions",
```

Comment:
@vpandiarajan20 Probably should name this `sentry-api-0-organization-sentry-functions` to be consistent